src/main/proto/netflix/titus/titus_job_api.proto

```diff
@@ -187,16 +187,28 @@ message JobDisruptionBudget {
         uint32 limit = 1;
     }
 
-    /// Maximum number of times a task can be evicted.
-    message EvictionLimit {
+    /// Maximum number of times a task can be relocated.
+    message RelocationLimit {
         uint32 limit = 1;
     }
 
     oneof Policy {
```

Comment:
Convert the concept to a rate or something that allows a number of terminations per time period?

Reply:
I have introduced a new oneof type `Rate`.